### PR TITLE
Update version numbers to 6.0.2

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Update version numbers to 6.0.2

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
@TravisEz13 Merge conflicts in packaging.psm1

**Comment by TravisEz13** [comment]
@adityapatwardhan  I've resolved the merge conflicts

File: tools/releaseTools.psm1 [hunk 1]
```diff
@@ -248,4 +248,54 @@ function Get-NewOfficalPackage
...
+    )
+
+    $metaDataPath = (Join-Path -Path $PSScriptRoot -ChildPath 'metadata.json')
+    $metaData = Get-Content -Path $metaDataPath | convertfrom-json
+    $currentTag = $metaData.ReleaseTag
+
+    $currentVersion = $currentTag -replace '^v'
+    $newVersion = $NewReleaseTag -replace '^v'
+    $metaData.NextReleaseTag = $NextReleaseTag
+    Set-Content -path $metaDataPath -Encoding ascii -Force -Value ($metaData | convertto-json)
```
Comment (daxian-dbw) [inline tools/releaseTools.psm1]: Please remind me again, is the `ReleaseTag` field in `metadata.json` always points to the latest GA release?

File: tools/releaseTools.psm1 [hunk 2]
```diff
@@ -248,4 +248,54 @@ function Get-NewOfficalPackage
...
+
+    Get-ChildItem -Path $Path -Recurse -File |
+        Where-Object {$_.Extension -notin '.icns','.svg' -and $_.NAME -ne 'CHANGELOG.md' -and $_.DirectoryName -notmatch '[\\/]docs|demos[\\/]'} |
```
Comment (daxian-dbw) [inline tools/releaseTools.psm1]: Just to confirm: we are not going to change version strings for docs from now on, right?

**Review by daxian-dbw — APPROVED** [review]

File: tools/releaseTools.psm1 (hunk 1 above)
Comment (TravisEz13) [reply tools/releaseTools.psm1]: filed #6405 for the issue I thought you were talking about.  the next step changes the release tag.  We can refine this behavior later.

**Comment by TravisEz13** [comment]
Windows and Linux CI have passed.  Merging so that docs will point to the latest release.

**Pull request merged**